python-client/docs/invocations.md

```diff
@@ -149,12 +149,12 @@ from multinode import Multinode
 
 mn = Multinode()
 
-@mn.function()
+@mn.function(cpu=4.0, memory="16 GiB")
 def run_subtask(y):
     out =  # ... perform calculation
     return out
 
-@mn.function()
+@mn.function(cpu=0.1, memory="1 GiB")
```

Comment:
Just above it reads:

> For example, this is useful for implementing distributed computations where:
> - the resource requirements change over different stages of the computation;
> - the parallelism strategy is determined at runtime.

The second point was demonstrated in this code block, but not the first.